README.md

```diff
@@ -34,6 +34,15 @@ Ensure that the two following urls / ports have disabled enhanced tracking prote
 - `localhost:3000`
 - `keycloak:8081`
 
+MAC OSX Development
+************************************************************************
+
+On MAC OS there are a few items to be cognizant of:
+- Matllab and the qldl python package do not seem to work on the M1 mac - so until this hase been fixed or these dependencies removed local dev on mac M1 is not possible.
```

Comment:
There is a small typo in "Matllab", it should be "Matlab". I also couldn't find a "qldl" package on PyPI, was it perhaps "pyqldb"?

Reply:
ah yes it's a confusing package name - it's qdldl https://github.com/osqp/qdldl-python p correcting